src/makefile

```diff
@@ -84,7 +84,7 @@ test: tests
 	LD_LIBRARY_PATH=. ./tests/opaque-munit$(EXT) --fatal-failures
 
 utils/opaque: utils/main.c
-	gcc $(CFLAGS) -I. -o utils/opaque utils/main.c -L. -lopaque -lsodium
+	gcc $(CFLAGS) -I. -o utils/opaque utils/main.c -L. $(LDFLAGS) -lopaque -lsodium
 
```

Comment:
i think i nailed it now with https://github.com/stef/libopaque/commit/e258d6a58f7dca3e8fc20aebe49f97bbd6373e95